cms/pytest.ini

```diff
@@ -12,6 +12,17 @@ filterwarnings =
     default
     ignore:No request passed to the backend, unable to rate-limit:UserWarning
     ignore::xblock.exceptions.FieldDataDeprecationWarning
+    # Remove next two default_app_config warnings after updating Django to 4.2
+    ignore:.*You can remove default_app_config.*:PendingDeprecationWarning
+    ignore:.*You can remove default_app_config.*:django.utils.deprecation.RemovedInDjango41Warning
```

Comment:
If I remember correctly, `RemovedInDjango41Warning` is a subclass of `PendingDeprecationWarning` in Django 3.2 (and a subclass of the `DeprecationWarning` in Django <4.2, but we don't use this version). Specifying `PendingDeprecationWarning` should be enough.

Reply:
i agree, i actually had this removed but forgot to commit it. will do so with the rebase